openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -1,23 +1,38 @@
 #include "openlr/openlr_simple_decoder.hpp"
 
+#include "routing/road_graph.hpp"
+
 #include "indexer/classificator_loader.hpp"
 #include "indexer/index.hpp"
 
 #include "platform/local_country_file.hpp"
 #include "platform/local_country_file_utils.hpp"
 #include "platform/platform.hpp"
 
+#include "openlr/decoded_path.hpp"
+#include "openlr/openlr_model.hpp"
+#include "openlr/openlr_model_xml.hpp"
+
 #include "coding/file_name_utils.hpp"
 
+#include "base/stl_helpers.hpp"
+
 #include "3party/gflags/src/gflags/gflags.h"
+#include "3party/pugixml/src/pugixml.hpp"
 
+#include <algorithm>
 #include <cstdint>
 #include <cstdio>
+#include <fstream>
 #include <string>
+#include <unordered_map>
 #include <vector>
 
 DEFINE_string(input, "", "Path to OpenLR file.");
 DEFINE_string(output, "output.txt", "Path to output file");
+DEFINE_bool(assessment_output, false,
```

Comment:
Why not to just keep two flags - the one for assessment tool output path, the other for the spark output? If the flag is not set, there is no need to write anything. Another way is to get rid of boolean flag and create a string flag, with possible values "spark" and "assessment".